setup.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 # Create and enable a virtual environment
-python -m venv --clear env
+python3 -m venv --clear env
```

Comment:
Looks like you've deleted some of setup.sh, so it no longer activates the virtual env
https://github.com/arjpaiva/DevOps-Course-Starter/commit/ff67230c5d20b70f4889c4c0156894b8d328b60c#diff-6aa367da3fdfc952b15def7759aa5f0b

Alternatively you have to run `source env/scripts/activate` yourself, which is fine but should be included in the README.